tests/integration/models/os.spec.coffee

```diff
@@ -121,12 +121,37 @@ describe 'OS model', ->
 				promise = balena.models.os.getSupportedVersions('raspberrypi')
 				m.chai.expect(promise).to.eventually.satisfy(areValidVersions)
 
+			it 'should cache the results', ->
+				balena.models.os.getSupportedVersions('raspberry-pi')
+				.then (result1) ->
```

Comment:
`result1` is also coming from cache, isn't it? There are calls for `raspberrypi` without clearing the cache before this test, which would set the cache. This still tests what is supposed to test, and I don't think it can result in flaky tests, just thought I should mention it in case you didn't take it into consideration.

Reply:
Yes that's true @sradevski ,
Maybe we can use a different device type, or call the `_clear...()` helper before.
Just give me a note and I can change it if you think we should do so.